pulumi/__main__.py

```
@@ -26,6 +26,9 @@
 container_port = config.get_int("containerPort")
 assert container_port, "containerPort must be set"
 
+bedingungsbaum_base_url = config.get("bedingungsbaumBaseUrl")
+assert bedingungsbaum_base_url, "bedingungsbaumBaseUrl must be set"
```

sehr gut, fail fast :)